internal/cel/cel.go

```diff
@@ -294,3 +294,31 @@ func (p *CELParser) CheckStepRunOutAgainstKnownV1(out *StepRunOut, knownType sql
 
 	return nil
 }
+
+func (p *CELParser) EvaluateBooleanExpression(expr string, inputData map[string]interface{}) (bool, error) {
```

Comment:
Wasn't sure if this was the right kind of thing to do or not, would love some ideas here!